kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -7663,7 +7663,7 @@ messages:
          }
 
          % Give newbies a little bonus.
-         if piBase_Max_health < 26 and random(20,bound(piBase_Max_health,20,25)) = 20
+         if piBase_Max_health = < Send(Send(SYS,@GetSettings),@GetPKillEnableHP)
```

Comment:
@Meridian59 We missed this `= <` and introduced a compilation error. I'm putting together a new PR.